### ISSUE TITLE
Trying to get property of non-object

### ISSUE THREAD

**Issue by perito77** [humhub/humhub#950]
Hi All,
First I want to thank you guys for creating this.
Sadly I require some help. I made a stupid mistake in pasting an markdown link in a post. Since then my dashboard and the main space are refusing to load. 
Other modules and spaces (where I haven't posted) seems to work fine...

error  Sun, 12 Apr 2015 21:16:39 -0500  php
Trying to get property of non-object (/var/www/intranet/protected/modules/polls/widgets/views/entry.php:81) Stack trace: #0 /var/www/intranet/protected/vendors/yii/web/widgets/CWidget.php(244): PollWallEntryWidget->renderFile() #1 /var/www/intranet/protected/modules/polls/widgets/PollWallEntryWidget.php(20): PollWallEntryWidget->render() #2 /var/www/intranet/protected/vendors/yii/web/CBaseController.php(167): PollWallEntryWidget->run() #3 /var/www/intranet/protected/modules/polls/models/Poll.php(218): SpaceController->widget() #4 /var/www/intranet/protected/modules_core/wall/Base============================================StreamAction.php(253): Poll->getWallOut() #5 /var/www/intranet/protected/vendors/yii/web/actions/CAction.php(76): ContentContainerStreamAction->run() #6 /var/www/intranet/protected/vendors/yii/web/CController.php(308): ContentContainerStreamAction->runWithParams() #7 /var/www/intranet/protected/vendors/yii/web/filters/CFilterChain.php(133): SpaceController->runAction() #8 /var/www/intranet/protected/vendors/yii/web/filters/CFilter.php(40): CFilterChain->run() #9 /var/www/intranet/protected/vendors/yii/web/CController.php(1145): CAccessControlFilter->filter() #10 /var/www/intranet/protected/vendors/yii/web/filters/CInlineFilter.php(58): SpaceController->filterAccessControl() #11 /var/www/intranet/protected/vendors/yii/web/filters/CFilterChain.php(130): CInlineFilter->filter() #12 /var/www/intranet/protected/vendors/yii/web/CController.php(291): CFilterChain->run() #13 /var/www/intranet/protected/vendors/yii/web/CController.php(265): SpaceController->runActionWithFilters() #14 /var/www/intranet/protected/vendors/yii/web/CWebApplication.php(282): SpaceController->run() #15 /var/www/intranet/protected/vendors/yii/web/CWebApplication.php(141): WebApplication->runController() #16 /var/www/intranet/protected/vendors/yii/base/CApplication.php(180): WebApplication->processRequest() #17 /var/www/intranet/index.php(39): WebApplication->run() REQUEST_URI=/intranet/index.php?r=space/space/stream&limit=4&filters=&sort=c&from=&mode=normal&sguid=c31d045c-63bb-4d47-8c01-c5642b72fd9a&CSRF_TOKEN=20f89d64f532d6b7d2f910d992b9adfd3917e142&_=1428891399534 in /var/www/intranet/protected/modules/polls/widgets/views/entry.php (81) in /var/www/intranet/protected/modules/polls/widgets/PollWallEntryWidget.php (20) in /var/www/intranet/protected/modules/polls/models/Poll.php (218) in /var/www/intranet/protected/modules_core/wall/BaseStreamAction.php (253) in /var/www/intranet/index.php (39)

**Comment by luke-** [comment]
Hmm, the error seems more related to the Polls modules.
Looks like an user voted and is deleted now...

I'll try to reproduce it, which HumHub version do you use?

**Comment by perito77** [comment]
Thanks for your quick response!

I user the latest version: **0.11.1**

**Comment by luke-** [comment]
Did you changed a user from Active to Not approved?

**Comment by perito77** [comment]
I use OpenLDAP for authentication.
I do recall creating a test user and notice that the user data remains after removing it from the LDAP
I did remove this user directly from the database (part of the cleanup)
Maybe that was the trigger?

**Comment by luke-** [comment]
Oh yes, this test user answered a poll and it's now broken.

You can try to check the poll_answer_user table - there must be an invalid/deleted user_id

**Comment by luke-** [comment]
Alternativly you can disable & enable the poll module on the affected space.
But you will loose all polls on this space.

**Comment by perito77** [comment]
Well that solve the problem with the main space...
Still have issues with my own dashboard
I log out and check with another user, the same problem

I created another admin user and started deleting the posts of the previous 2 admins
**I did forget to mention. I did rename the name of the main admin. Maybe also this was a trigger?**

In the database a lot of post had NULL as data. I also deleted those.

**UPDATE**
** it seems changing the details of the main admin was the trigger**
